eo-runtime/src/test/eo/org/eolang/string-tests.eo

```diff
@@ -116,18 +116,6 @@
     $.equal-to
       "hello".as-hash
 
-[] > compares-string-with-nan-and-infinites
```

Comment:
@maxonfjvipon Why have you decided to remove that test?

Reply:
@maxonfjvipon NaN and infinites has unique bytes representation and it seems there is no such string that has the same bytes as NaN of infinities

Reply:
@maxonfjvipon Maybe we can show that `positive-infinity != any_string`?

Reply:
@volodya-lombrozo do you mean something like that?
```
[] > compares-string-with-positive-infinity
  assert-that > @
    "друг"
    $.not
      $.equal-to positive-infinity
```

Reply:
@maxonfjvipon Yes. As I understand the idea behind that test was almost the same.